### PR TITLE
changed db to covid19 and added global test script

### PR DESCRIPTION
**Pull request by attwad**
Also added a dev script to run all tests instead of having to change directory and run npm test in all of them.

### CODE REVIEW

File: dev/test_all.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+npm --prefix=../verification/curator-service/api/ run-script test-silent
```
Comment (khmoran) [inline dev/test_all.sh:3]: why silent?

File: data-serving/data-service/package.json [hunk 2]
```diff
@@ -11,12 +11,13 @@
   "scripts": {
     "build": "tsc",
     "dev": "nodemon",
-    "import-data": "mongoimport --db dev --collection cases --file ../samples/line-list.json",
+    "import-data": "mongoimport --db covid19 --collection cases --file ../samples/line-list.json",
```
Comment (khmoran) [inline data-serving/data-service/package.json:14]: how are we going to handle dev/prod instances down the road? obviously this is fine locally but when we're deploying to both prod and non-prod envs in AWS, will we name our dbs in Atlas something like covid19-dev and covid19-prod?

**Comment by khmoran** [comment]
Is anyone using the db in Atlas right now? I can create new ones to reflect this change

File: dev/test_all.sh (hunk 1 above)
Comment (attwad) [reply dev/test_all.sh:3]: This script is aimed at being just a quick && simple check that I didn't break too many things, I wanted the output to be minimal (no per-case log) but that still prints errors if there are any. Let me know if you want me to drop the silent, it just suited my usecase.

File: data-serving/data-service/package.json (hunk 2 above)
Comment (attwad) [reply data-serving/data-service/package.json:14]: We could keep the same db name between dev and prod, the "instances" will be different, that script currently connects to the default local DB and can't upload to atlas for example.

Or we could separate them yes and have covid19-dev || dev-covid-19, I have no strong preference.

**Comment by attwad** [comment]
I haven't used atlas so far, feel free to rename/recreate the DB

File: dev/test_all.sh (hunk 1 above)
Comment (khmoran) [reply dev/test_all.sh:3]: gotcha, sg, the rest of us can always add a more verbose one if we want it :D

File: data-serving/data-service/package.json (hunk 2 above)
Comment (khmoran) [reply data-serving/data-service/package.json:14]: hmm... separate instances would probably be cleaner (don't have to change the db names across envs) -- maybe different sharded clusters for each? https://docs.atlas.mongodb.com/reference/glossary/#term-sharded-cluster

anyway, we can decide later once we start setting up CI/CD

File: verification/curator-service/api/src/util/validate-env.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export default function validateEnv(): Readonly<{
         }),
         DB_CONNECTION_STRING: str({
             desc: 'MongoDB URI provided to MongoClient.',
-            devDefault: 'mongodb://localhost:27017/dev',
```
Comment (axmb) [inline verification/curator-service/api/src/util/validate-env.ts:19]: There's probably an analogous default to update in data-service.

Comment (attwad) [reply verification/curator-service/api/src/util/validate-env.ts:19]: good catch, done.

**Pull request merged**